Script/AtomicEditor/ui/frames/menus/MainFrameMenu.ts

```diff
@@ -40,7 +40,8 @@ class MainFrameMenu extends Atomic.ScriptObject {
         MenuItemSources.createMenuItemSource("menu tools", toolsItems);
         MenuItemSources.createMenuItemSource("menu developer", developerItems);
         MenuItemSources.createMenuItemSource("menu help", helpItems);
-
+        this.goScreenshot = 0;
+        this.subscribeToEvent(Atomic.UpdateEvent((ev) => this.handleScreenshot(ev)));
```

Comment:
OK, using the TBmessage subsystem is a **bad** idea. However, after looking at the AtomicEventLoop, this too seems like overkill, and all I may need is `setTimeout ( expression, timeout )` to do the needed job, I'll test it in a little while to see if it works.